Use two-letter language codes in RSS feeds, where available

redrun45 opened the pull request:
Evidently, RSS feeds are supposed to use the two-letter language codes, instead of three.  Switch to using two-letter codes where feasible:
https://forum.librivox.org/viewtopic.php?p=2479460#p2479460

A couple notes on tests.  Evidently, we had a stub RSS controller file sitting around, and our test harness preferred to send requests to that, instead of to the real one.  Also, our test harness includes little debug messages both before and after the response itself, so I had to do a little cleanup in order to validate the XML.  That validation might be a standard part of any new tests I write going forward...

redrun45 committed:
Switch to two-letter lang code in RSS

notartom commented on application/tests/controllers/rss/Rss_test.php:
```diff
@@ -0,0 +1,83 @@
+      $this->assertStringContainsString($str, $response);
+    }
+
+    // Ugly: our test harness inserts messages like "got an instance" and "called the actual controller", before and after the actual RSS/XML response document.  Trim them off.
```
Do we know where these come form? A quick grep of our own code base doesn't turn up anything, and searching on GitHub the contents of phpunit and ci-phpunit-test doesn't find anything here. I'd much rather we get rid of these at the source if at all possible than working around them with hacks.

(But thanks for the code comment, I would have been left wondering otherwise).

redrun45 committed:
Add basic tests for RSS
Also removed a redundant RSS controller, which "swallowed up" the testing calls.
